proto/osmosis/concentrated-liquidity/pool-model/query.proto

```diff
@@ -136,3 +139,18 @@ message LiquidityDepth {
     (gogoproto.nullable) = false
   ];
 }
+
+// ===================== MsgQueryClaimableFees
+message QueryClaimableFeesRequest {
```

Comment:
Looks like all CL queries are not generated correctly. The goal is to use `make run-querygen` for code-gening them.

Maybe we do this in a separate PR?

Reply:
Created separate issue for this https://github.com/osmosis-labs/osmosis/issues/4520